internal/util/util.go

```diff
@@ -112,3 +113,18 @@ func WrapString(s string, lim uint) string {
 
 	return buf.String()
 }
+
+
+// ReadLine will read a single line from a bufio.Reader
+func ReadLine(r *bufio.Reader) (string, error) {
+	var (
+		isPrefix bool
+		err error
+		line, ln []byte
+	)
```

Comment:
I think you can write more compact code using `:=` below, or I'm missing why you're not using it.

Reply:
Not sure what you are suggesting. Can you provide a diff?

Reply:
No, I can't, it seems the code cannot be simplified further.